### ISSUE TITLE
When using talkback screen reader, swiping menu items always closes the menu

### ISSUE THREAD

**Issue by morbidCode** [readest/readest#3035]
## Bug Description

For a bit of context, I’m a visually impaired user and I use TalkBack on Android. I ran into an issue that makes it a bit tough when navigating with a screen reader: When I open a menu (like Settings), swiping through the items always closes the menu. Makes it pretty hard to scroll through options because I have to guess where the items are since I can't swipe.

## Expected behavior

Talkback uses the swipe gesture to navigate through the screen. When swiping, it should be able to read  items and interact with them. Maybe if the app detects that a screen reader is turned on, it should not close the menu?

## Technical inputs

Operating System: Android 16
Readest Version: 0.9.98

My guess is the normal swipe gesture really should close menus. Maybe we can let the app bypass that normal function if it detects a screen reader?

**Comment by chrox** [comment]
Thanks for your feedback we will look into this later.
